app/controllers/stories_controller.rb

```diff
@@ -85,6 +85,13 @@ def create
     end
   end
 
+  def history
+    @story = policy_scope(Story).try(:find, params[:story_id])
```

Comment:
Sure, I will move it to a new ActivitiesController. :)

Oh, I got it. When it returns nil, having a`try` would prevent an 500 error `Unable to find policy of nil`. It would not be better to check if it is nil and return a 400 error? What do you guys think?